docs/configuring-output-after.asciidoc

```diff
@@ -21,6 +21,9 @@ Example:
 [source,yaml]
 ------------------------------------------------------------------------------
 fields: {project: "myproject", instance-id: "574734885120952459"}
+#-------------------------- Elasticsearch output --------------------------
+output.elasticsearch:
+  hosts: ["localhost:9200"]
```

Comment:
With the suggestion above, I'm not sure a change is needed here. What do you think @renshuki ?

Reply:
If you think the additional content is helpful, we can keep it in the example, but should specify that this example is for the ES output. Something like:

```
Example using the Elasticsearch output:

[source,yaml]
------------------------------------------------------------------------------
fields: {project: "myproject", instance-id: "574734885120952459"}
output.elasticsearch:
  hosts: ["localhost:9200"]
------------------------------------------------------------------------------
```

Reply:
I just want to make sure users understand they have to put the `fields` setting outside of the `output`. I think that adding:
> Example using the Elasticsearch output:

for clarification is the best option. Thanks.

Reply:
Great. That sounds good to me.

Reply:
Appreciate the help, I added the change.